main.py

```
@@ -3,11 +3,53 @@
+
+
+async def main() -> None:
     """Função principal da aplicação."""
-    pass
+
+    app = QApplication(sys.argv)
+    loop = QEventLoop(app)
+    asyncio.set_event_loop(loop)
+
+    bus = get_event_bus()
+    asyncio.create_task(bus.start())
```

Ruff (0.12.2)

Store a reference to the return value of `asyncio.create_task`

(RUF006)